.drone.yml

```diff
@@ -64,3 +64,24 @@ pipeline:
     when:
       branch: master
       event: push
+
+  deploy_production_cronjobs:
+    group: deploy
+    image: quay.io/mongodb/drone-helm:v2.14.1-0.1.0
+    # The release name should be unique across the namespace, the app or repo name is recommended
+    release: selected-tests
+    namespace: server-tig
+    environment:
+      - API_SERVER=https://api.prod.corp.mongodb.com
+    prefix: prod
+    secrets: [ prod_kubernetes_token ]
+    helm_repos: mongodb=https://10gen.github.io/helm-charts
+    chart: mongodb/cronjobs
+    chart_version: 1.3.0
+    tiller_ns: server-tig
+    client_only: true
+    values: "image.tag=git-${DRONE_COMMIT_SHA:0:7},image.repository=795250896452.dkr.ecr.us-east-1.amazonaws.com/server-tig/${DRONE_REPO_NAME},ingress.enabled=true,ingress.hosts[0]=selected-tests.server-tig.prod.corp.mongodb.com"
```

Comment:
Do these cron jobs need an ingress? Are they being hit by something else?

Reply:
You're right, I don't see an ingress in their example: https://github.com/10gen/kanopy-docs/blob/master/examples/cronjobs/.drone.yml. Will remove.